fearless/Modules/Staking/ValidatorInfo/ValidatorInfoViewController.swift

```diff
@@ -43,74 +18,90 @@ final class ValidatorInfoViewController: UIViewController {
     }
 
     private func configureTableView() {
-        // Cell for validator account display
         tableView.register(
             UINib(resource: R.nib.validatorInfoAccountCell),
             forCellReuseIdentifier: R.reuseIdentifier.validatorAccountCellId.identifier
         )
 
         tableView.register(
-            UINib(resource: R.nib.validatorInfoInformationCell),
-            forCellReuseIdentifier: R.reuseIdentifier.validatorInfoInformationCellId.identifier
+            UINib(resource: R.nib.validatorInfoCell),
+            forCellReuseIdentifier: R.reuseIdentifier.validatorInfoCellId.identifier
         )
 
-        tableView.register(
-            UINib(resource: R.nib.validatorInfoTitleSubtitleCell),
-            forCellReuseIdentifier: R.reuseIdentifier.validatorInfoTitleSubtitleCellId.identifier
-        )
-
-        tableView.register(
-            UINib(resource: R.nib.validatorInfoWebCell),
-            forCellReuseIdentifier: R.reuseIdentifier.validatorInfoWebCellId.identifier
-        )
+        tableView.registerClassForCell(ValidatorInfoEmptyStakeCell.self)
 
         tableView.alwaysBounceVertical = false
     }
 }
 
 extension ValidatorInfoViewController: UITableViewDelegate {
-    func tableView(_: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        guard indexPath.section > 0 else { return RowType.accountRowHeight }
+    func tableView(_: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        switch viewModel[section] {
+        case .account:
+            return CGFloat.leastNormalMagnitude
+        default:
+            return ValidatorInfoViewModel.headerHeight
+        }
+    }
 
-        return RowType.rowHeight
+    func tableView(_: UITableView, heightForFooterInSection _: Int) -> CGFloat {
+        CGFloat.leastNormalMagnitude
+    }
+
+    func tableView(_: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
```

Comment:
non AutoLayout cells?

Reply:
Nope, but don't see any problems with fixed height cells: more code to write but it already has been written :)